### ISSUE TITLE
ONYX-966: Internal error when comments button pressed in the interview page.

### ISSUE THREAD

**Issue by ymarcon** [obiba/onyx#971]
Jira issue originally created by user @dwainelson:

The error goes away if you add "action.EXECUTE.Interrupted.ruby.BloodSamplesCollection" to the message_en.properties file but I am not sure this is the correct solution. I have alse seen this error for the UrineSamplesCollection. 

A DB for the Ontario config is attached.

Log Excerpt:
2009-10-21 18:03:03,000 - INFO  - resource.PropertiesFactory - Loading properties files from jar:file:/C:/Program%20Files/Apache%20Software%20Foundation/Tomcat%206.0/webapps/ohs-webapp/WEB-INF/lib/obiba-wicket-1.0.5.jar!/org/obiba/wicket/markup/html/table/AjaxDataTable.properties
2009-10-21 18:03:28,625 - INFO  - efaultInterviewManagerImpl - User administrator has locked interview for participant 5000000.
2009-10-21 18:03:29,390 - INFO  - resource.PropertiesFactory - Loading properties files from file:/C:/Program%20Files/Apache%20Software%20Foundation/Tomcat%206.0/webapps/ohs-webapp/WEB-INF/classes/org/obiba/onyx/webapp/participant/page/InterviewPage.properties
2009-10-21 18:03:29,437 - INFO  - resource.PropertiesFactory - Loading properties files from file:/C:/Program%20Files/Apache%20Software%20Foundation/Tomcat%206.0/webapps/ohs-webapp/WEB-INF/classes/org/obiba/onyx/webapp/stage/panel/StageSelectionPanel.properties
2009-10-21 18:03:31,906 - INFO  - resource.PropertiesFactory - Loading properties files from jar:file:/C:/Program%20Files/Apache%20Software%20Foundation/Tomcat%206.0/webapps/ohs-webapp/WEB-INF/lib/onyx-core-1.5.4.jar!/org/obiba/onyx/wicket/reusable/Dialog.properties
2009-10-21 18:03:31,906 - INFO  - resource.PropertiesFactory - Loading properties files from file:/C:/Program%20Files/Apache%20Software%20Foundation/Tomcat%206.0/webapps/ohs-webapp/WEB-INF/classes/org/obiba/onyx/webapp/action/panel/InterviewLogPanel.properties
2009-10-21 18:03:31,921 - ERROR - apache.wicket.RequestCycle - Exception in rendering component: [Component id = action]
org.apache.wicket.WicketRuntimeException: Exception in rendering component: [Component id = action]
	at org.apache.wicket.Component.renderComponent(Component.java:2519)
	at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:60)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.markup.html.panel.Fragment.renderFragment(Fragment.java:278)
	at org.apache.wicket.markup.html.panel.Fragment.onComponentTagBody(Fragment.java:206)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.markup.html.list.Loop.renderItem(Loop.java:201)
	at org.apache.wicket.markup.html.list.Loop.renderChild(Loop.java:190)
	at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.markup.html.form.Form.onComponentTagBody(Form.java:1710)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.markup.html.form.Form.onRender(Form.java:1774)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.Component.renderComponent(Component.java:2420)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:785)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
	at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
	at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
	at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
	at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
	at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
	at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
	at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:200)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.springframework.context.NoSuchMessageException: No message found under code 'action.EXECUTE.Interrupted.ruby.BloodSamplesCollection' for locale 'en'.
	at org.springframework.context.support.AbstractMessageSource.getMessage(AbstractMessageSource.java:161)
	at org.springframework.context.support.DelegatingMessageSource.getMessage(DelegatingMessageSource.java:71)
	at org.springframework.context.support.AbstractApplicationContext.getMessage(AbstractApplicationContext.java:987)
	at org.obiba.wicket.model.MessageSourceResolvableStringModel.getObject(MessageSourceResolvableStringModel.java:70)
	at org.apache.wicket.Component.getModelObject(Component.java:1558)
	at org.apache.wicket.Component.getModelObjectAsString(Component.java:1580)
	at org.apache.wicket.markup.html.basic.Label.onComponentTagBody(Label.java:111)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	... 81 more
2009-10-21 18:03:35,046 - INFO  - $MenuItemSelectionBehavior - linkPageClass = HomePage, pageClass = HomePage
2009-10-21 18:03:35,046 - INFO  - $MenuItemSelectionBehavior - linkPageClass = ParticipantSearchPage, pageClass = HomePage
